### PR TITLE
Spanish talk

### PR DESCRIPTION
**Pull request by JuanCaicedo**

### CODE REVIEW

File: content/first-talk-in-spanish.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: My first talk in Spanish
+---
+# My first talk in Spanish
+This week I'll be giving a talk at NodeConf Argentina. It's going to be my first
+time giving a talk in Spanish. That's a huge deal for me, and it's all thanks to
+my friend Raul, thanks to whom I've come to see this as a responsibility I have
+to the latin american development community
```
Comment (rosston) [inline content/first-talk-in-spanish.mdx]: This sentence flows kind of weird. I wonder if you could split it up. Maybe something like the following? (ignore/edit as desired)
>That's a huge deal for me, and it's all thanks to my friend Raul. Thanks to him, I've come to see giving a tech talk in Spanish as a responsibility I have to the Latin American development community.

(I also capitalized "latin american" and added a period at the end.)

File: content/first-talk-in-spanish.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Venezuelan, we had a lot in common to talk about and became friends quickly.
+
+The conference offered to let us do our talks in Spanish or in English.
+
+I'm comfortable speaking Spanish, but I'm most used to talking about technical
+topics in English. I didn't feel confident talking about programming in my
+native language, so I chose to give my talk in English.
+
+Raul had left Caracas to work in Chile when things started to get very bad under
+Maduro's presidency. He had been working hard to build a new life for him and
```
Comment (rosston) [inline content/first-talk-in-spanish.mdx]: I'd suggest using "himself" in place of "him", ie:
>He had been working hard to build a new life for _himself_ and...

File: content/first-talk-in-spanish.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+his wife in a completely new country, all while caring for his family back home.
+He really wanted to get exposure for the topic he was speaking on, so he also
+gave his talk in English.
+
+Here's a [link]() to my talk from back then. I would love to give you one for
+Raul's talk, but unfortunately halfway through his presentation the lights went
+out and the footage was lost. (It was a great talk)
```
Comment (rosston) [inline content/first-talk-in-spanish.mdx]: Should probably add a period after "talk".

File: content/first-talk-in-spanish.mdx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+# NodeConf AR
+The opportunity to give a talk at this conference is actually my second time in
+the city of Buenos Aires. I came to this same conference in November 2016.
+
+I had seen on twitter that Raul had gotten accepted to give a talk at the
```
Comment (rosston) [inline content/first-talk-in-spanish.mdx]: English, so "twitter" should be capitalized. 🙂

File: content/first-talk-in-spanish.mdx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+NodeConf that year. When I sent him a tweet to congratulate him, he asked "Are
+you going to come?" I couldn't think of any good reasons to say no, so I decided
+I would fly out to attend.
+
+Raul gave an amazing talk about the [Ethereum platform](), this time in Spanish.
+What he told me is that he now thought it was important to make technical
+content available in Spanish, because technology skills are the best thing to
+lift people Latin America out of poverty.
```
Comment (rosston) [inline content/first-talk-in-spanish.mdx]: I think you're missing a word:
>lift people _in_ Latin America

File: content/first-talk-in-spanish.mdx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+# Privilege and giving back
+Raul inspired me. Since that day, giving a talk in Spanish has one of my goals.
```
Comment (rosston) [inline content/first-talk-in-spanish.mdx]: I think you're missing a word:
>giving a talk in Spanish has _been_ one of my goals

File: content/first-talk-in-spanish.mdx [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+Nowadays, I live in Colombia and work remotely. It has become apparent to me all
+the privileges I've had growing up as an immigrant in the United States. Someone
+who lived in Colombia through the turbulent 1990s and 2000s grew up in a very
+different way from me, and because of that had fewer opportunities available to
+them than I did.
+
+Sure, I'm a self taught myself programmer and worked hard to break into the
```
Comment (rosston) [inline content/first-talk-in-spanish.mdx]: >I'm a self taught myself programmer

I think you have an extra word here. Probably just remove "myself"?

**Comment by JuanCaicedo** [comment]
Thanks for the edits @rosston

**Pull request merged**